app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -109,4 +109,10 @@ def interesting_klasses(obj)
 
     classes.where(:id => klasses)
   end
+
+  def explicit_value?(field)
+    return true if params[:action] == 'clone'
+    return unless params[:host]
```

Comment:
Nitpick, return false instead of nil for consistency with the other possible bool return values.  (Ditto in the other method)